### PR TITLE
Make reply preview not an overlay so you can see new messages

### PR DESCRIPTION
**Pull request by bwindels**
Came up during the polish call with @niquewoodhouse 

reply-no-overlay2

### CODE REVIEW

**Comment by t3chguy** [comment]
The composer buttons getting moved up looks really broken

**Comment by bwindels** [comment]
> The composer buttons getting moved up looks really broken

This should be fixed now:

reply-no-overlay3

**Comment by bwindels** [comment]
(@niquewoodhouse has already approved the design OOB)

**Comment by t3chguy** [comment]
Should autocomplete panes get the same treatment?

File: src/components/views/rooms/MessageComposer.js [hunk 1]
```diff
@@ -222,7 +223,24 @@ export default class MessageComposer extends React.Component {
...
+            // that the ScrollPanel listening to the resizeNotifier can
+            // correctly measure it's new height and scroll down to keep
+            // at the bottom if it already is
+            setTimeout(() => {
+                this.props.resizeNotifier.notifyTimelineHeightChanged();
+            }, 100);
+        }
+    };
+
+    componentWillUnmount() {
```
Comment (turt2live) [inline src/components/views/rooms/MessageComposer.js:238]: linter says this is duplicated

**Comment by bwindels** [comment]
> Should autocomplete panes get the same treatment?

Good idea, but not sure I want to put in the time now, sorry.

**Pull request merged**